### PR TITLE
Adding UCB1 functionality.

### PR DESCRIPTION
**Pull request by wengzilla**

### CODE REVIEW

File: lib/bandit/players/ucb.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    def choose_alternative(experiment)
+      best_alternative(experiment)
+    end
+
+    def best_alternative(experiment)
+      memoize(experiment.name) {
+        best = nil
+        best_rate = nil
+        experiment.alternatives.each { |alt|
+          rate = experiment.conversion_rate(alt) + confidence_interval
```
Comment (bmuller) [inline lib/bandit/players/ucb.rb]: Shouldn't the line above include some params passed to confidence_interval?

Comment (wengzilla) [inline lib/bandit/players/ucb.rb]: You're definitely right... I'll need to go in and fix that.

**Comment by bmuller** [comment]
Oh - if you get a chance - a brief description in players.rdoc would be awesome.

Thanks!

**Comment by wengzilla** [comment]
Rebased and fixed as per your suggestions.

**Comment by bmuller** [comment]
Merged in 2bb9096.  Thanks!

**Pull request closed without merging**